packages/components/addon/components/hds/table/th-button-sort.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+const NOOP = () => {};
+
+export default class HdsTableThButtonSortComponent extends Component {
+  /**
+   * @param onClick
+   * @type {function}
+   * @default () => {}
+   */
+  get onClick() {
+    let { onClick } = this.args;
+    return onClick || NOOP;
```

Comment:
In other places we check for this argument to be a function, we may want to consider using the same pattern for consistency.
```suggestion
    if (typeof onClick === 'function') {
      return onClick;
    } else {
      return NOOP;
    }
```

Reply:
good point. done in https://github.com/hashicorp/design-system/pull/1860/commits/4f3bb289ae280f0ac2cfbd9c95e2309cd5244322